web/service/src/main/java/org/mobilitydata/gtfsvalidator/web/service/controller/ValidationController.java

```diff
@@ -121,6 +127,22 @@ public ResponseEntity runValidator(
     }
   }
 
+  @GetMapping("/version")
+  public ResponseEntity currentVersion() {
+    // https://gtfs-validator-web-mbzoxaljzq-ue.a.run.app/version
+    StringBuilder stringBuilder = new StringBuilder();
+    VersionResolver checker = new VersionResolver();
+    VersionInfo versionInfo = checker.getVersionInfoWithTimeout(TIMEOUT);
```

Comment:
If we are only interested in the current version, I suggest instead of using `getVersionInfoWithTimeout` make the [VersionResolver#resolveCurrentVersion](https://github.com/MobilityData/gtfs-validator/blob/ccfdd65b08187f58caed823c7765366de1abebbd/main/src/main/java/org/mobilitydata/gtfsvalidator/util/VersionResolver.java#L114) method public and call it directly. This will save a network call that is executed inside `getVersionInfoWithTimeout`.